scripts/obtain/dotnet-install.ps1

```diff
@@ -325,6 +330,9 @@ function Get-Download-Link([string]$AzureFeed, [string]$SpecificVersion, [string
     elseif ($Runtime -eq "aspnetcore") {
         $PayloadURL = "$AzureFeed/aspnetcore/Runtime/$SpecificVersion/aspnetcore-runtime-$SpecificVersion-win-$CLIArchitecture.zip"
     }
+    elseif ($Runtime -eq "windowsdesktop") {
+        $PayloadURL = "$AzureFeed/Runtime/$SpecificVersion/windowsdesktop-runtime-$SpecificVersion-win-$CLIArchitecture.zip"
```

Comment:
Are there any precautions we need to take to prepare for this sharedfx moving to a different repo in the future, which means it will need a different base path than `Runtime`? (https://github.com/dotnet/core-setup/issues/5732)

Reply:
The scripts rely heavily on the predictability of the 'path' to the archive. Open question: is it possible to have two path to the same blob storage location?

Reply:
Not that I know of. 😕 https://stackoverflow.com/q/7641316

Core-Setup *could* upload to two virtual dirs for now, and WD would eventually take over the new one. Not great, of course.

Is it feasible to have this script look at some theoretical WD location, and fall back to the NETCore location when it 404's?

Reply:
@dagood 
I'm trying to hold the line on theoretical lookups - we can do 2 theoretical lookups right now.